packages/lib/server/rateLimit.ts

```diff
@@ -0,0 +1,21 @@
+import type { NextApiHandler } from "next";
+
+import { HttpError } from "../http-error";
+import { rateLimiter } from "../rateLimit";
+
+export const rateLimitHandler: NextApiHandler = async (req) => {
```

Comment:
Why did we need to build this rate limiting logic? Don't we already have this and the calculations are done for us?